docs/en/operations/settings/settings.md

```diff
@@ -9221,7 +9221,7 @@ Type: Bool
 
 Default value: 1
 
-Allows or forbids empty INSERTs, enabled by default (throws an error on an empty insert)
+Allows or forbids empty INSERTs, enabled by default (throws an error on an empty insert). Only applies to inserts using [`clickhouse-client`](docs/en/interfaces/cli).
```

Comment:
It also applies to GRPC server inserts (I didn't even know we have that though), and it only works on client side, not on server side. So it means those inserts won't reach the server. 
```suggestion
Allows or forbids empty INSERTs, enabled by default (throws an error on an empty insert). Only applies to inserts using [`clickhouse-client`](docs/en/interfaces/cli) (on the client side) and INSERTs on the [gRPC interface](docs/en/interfaces/grpc).
```

Or something along these lines.

Reply:
Thanks, I added gRPC.